internal/conventionalcommits/conventional_commits_test.go

```diff
@@ -453,6 +453,57 @@ END_COMMIT_OVERRIDE`,
 				},
 			},
 		},
+		{
+			name: "parse_multiple_lines_message_inside_nested_commit_multi_line_headers",
```

Comment:
Underscore is preferred over whitespace in test names: https://g3doc.corp.google.com/company/teams/go-community/gotip/episodes/117.md?cl=head